src/models/event.js

```diff
@@ -226,7 +226,12 @@ utils.extend(module.exports.MatrixEvent.prototype, {
      * @return {Object} The event content JSON, or an empty object.
      */
     getContent: function() {
-        return this._clearEvent.content || this.event.content || {};
+        if (this._replacingEvent && !this.isRedacted()) {
```

Comment:
Why the redacted check here...?

Reply:
Because we don't want to return any replaced content for a redacted event. I guess a better way to do this would be to set `this._replacingEvent` to `null` in `makeRedacted`.